distribution/docs/src/main/resources/content/_sources/solr-catalog-provider.adoc

```diff
@@ -47,7 +47,7 @@ To enable TLS:
  ----
  solr.client=HttpSolrClient
  solr.http.port=8994
- solr.http.url=http://localhost:${solr.http.port}/solr
+ solr.http.url=http://localhost:_DO_NOT_EXPAND_${solr.http.port}/solr
```

Comment:
No `_DO_NOT_EXPAND_ here in the doc. Just in the file at build time to let the build process to not expand the variable and instead leave it as is to be expanded on the installed system.